sdcm/tester.py

```diff
@@ -589,7 +591,7 @@ def argus_collect_gemini_results(self):
             self.log.warning("Error submitting gemini results to argus", exc_info=True)
 
     def collect_ssl_conf(self):
-        shutil.copytree(Path(get_data_dir_path('ssl_conf')), Path(self.logdir) / 'ssl_conf')
+        shutil.copytree(Path(get_data_dir_path('ssl_conf')), Path(self.logdir) / 'ssl_conf', dirs_exist_ok=True)
```

Comment:
why is it now required?

Reply:
it's probably not needed, it was in the why while working on refactoring `test_tester.py`